PSReadLine/Completion.cs

```diff
@@ -372,9 +372,15 @@ private void DoReplacementForCompletion(CompletionResult completionResult, Comma
         {
             var replacementText = completionResult.CompletionText;
             int cursorAdjustment = 0;
+            CompletionResultType[] selfContainedCompletionResultTypes = {CompletionResultType.Command,
```

Comment:
This should be a static array so you don't create a new array for every completion.